test/parallel/test-util-emit-experimental-warning.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+// Flags: --expose-internals
+const common = require('../common');
+const assert = require('assert');
+const { emitExperimentalWarning } = require('internal/util');
+
+// This test ensures that the emitExperimentalWarning in internal/util emits a
+// warning when passed a unsupported feature and that it simply returns
+// when passed the same feature multiple times.
+
+process.on('warning', common.mustCall((warning) => {
+  assert(/is an experimental feature/.test(warning.message));
+}, 2));
+
+emitExperimentalWarning('feature1');
+emitExperimentalWarning('feature1');
+emitExperimentalWarning('feature2'); // should not warn
```

Comment:
Seems like the comment is on the wrong line / the numbers got mixed up?

Reply:
@BridgeAR : You're right.. my bad. I've fixed it now. Kindly review. Thanks !